app/packages/utilities/src/index.ts

```diff
@@ -380,6 +399,85 @@ export const CLIPS_FRAME_FIELDS = withPath(LABELS_PATH, [
   "Polylines",
 ]);
 
+export const DISABLED_PATHS = ["id", "filepath", "tags", "metadata"];
+export const DISABLED_FIELD_TYPES = [
+  OBJECT_ID_FIELD,
+  FRAME_NUMBER_FIELD,
+  FRAME_SUPPORT_FIELD,
```

Comment:
I can imagine users defining custom fields with these types. The expectation is that these are always disabled, though?

Reply:
valid point. following up with Brian to see refresh my memory on this.